hazelcast-client-legacy/src/test/java/com/hazelcast/client/listeners/AbstractListenersOnReconnectTest.java

```diff
@@ -72,7 +73,9 @@ public void run() throws Exception {
         assertTrueEventually(new AssertTask() {
             @Override
             public void run() throws Exception {
-                assertEquals(EVENT_COUNT, eventCount.get());
+                int current = eventCount.get();
+                Logger.getLogger(this.getClass()).info("[testListenersInternal] Current event count is " + current);
+                assertEquals(EVENT_COUNT, current);
```

Comment:
Pass the error message to logger as parameter. This way when test failed, you will have the error message printed.